htdocs/js/jquery.fileupload.js

```diff
@@ -209,10 +213,13 @@ $(function() {
         $.extend( image, data );
         $field.data( "image-attributes", image );
 
+        var escape_titletext = image.title.replace( /</g, '&lt;' ).replace( /'/g, "&apos;" );
```

Comment:
I've got an idea, but I don't know if it's elegant. Instead of manipulating text, for which the functions don't already exist, you'd have to manipulate the DOM so that the page actually contains the img you're trying to build up, non-displayed. Then ask jQuery for the html source of that img.